src/openclaw_pipeline/commands/watch_progress.py

```
@@ -0,0 +1,222 @@
+    if not line:
+        return None
+    try:
+        return json.loads(line)
+    except json.JSONDecodeError:
+        return None
+
+
+def detect_openclaw_process_lines(vault_dir: Path) -> list[str]:
+    try:
+        result = subprocess.run(
+            ["ps", "aux"],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)